crates/router/src/connector/netcetera/transformers.rs

```diff
@@ -125,6 +122,68 @@ impl
     }
 }
 
+impl
+    TryFrom<
+        types::ResponseRouterData<
+            api::Authentication,
+            NetceteraAuthenticationResponse,
+            types::authentication::ConnectorAuthenticationRequestData,
+            types::authentication::AuthenticationResponseData,
+        >,
+    > for types::authentication::ConnectorAuthenticationRouterData
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: types::ResponseRouterData<
+            api::Authentication,
+            NetceteraAuthenticationResponse,
+            types::authentication::ConnectorAuthenticationRequestData,
+            types::authentication::AuthenticationResponseData,
+        >,
+    ) -> Result<Self, Self::Error> {
+        let response = match item.response {
+            NetceteraAuthenticationResponse::Success(response) => {
+                let authn_flow_type = match response.acs_challenge_mandated {
+                    Some(ACSChallengeMandatedIndicator::Y) => {
+                        types::authentication::AuthNFlowType::Challenge(Box::new(
+                            types::authentication::ChallengeParams {
+                                acs_url: response.authentication_response.acs_url.clone(),
+                                challenge_request: response.encoded_challenge_request,
+                                acs_reference_number: response
+                                    .authentication_response
+                                    .acs_reference_number,
+                                acs_trans_id: response.authentication_response.acs_trans_id,
+                                three_dsserver_trans_id: Some(response.three_ds_server_trans_id),
+                                acs_signed_content: None,
+                            },
+                        ))
+                    }
+                    _ => types::authentication::AuthNFlowType::Frictionless,
```

Comment:
Remove this default so that in future type safe errors can be detected at compile time